src/internal/m365/collection/drive/library_handler.go

```diff
@@ -17,10 +18,54 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api"
 )
 
+type baseLibraryHandler struct {
```

Comment:
Similar request here with naming.
```suggestion
type baseSiteDriveHandler struct {
```